repository/repo.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Collection } from 'mongoose';
+
+mongoose.set('strictQuery', false);
+const mongoDB = "mongodb://127.0.0.1:27017/test"; 
+
+async function main() {
+  await mongoose.connect(mongoDB);  
+};
+
+main().catch(err => console.log(err));
+const SoldierSchema = new mongoose.Schema({
+  _id:{type:String,required:true},
```

Comment:
Use eslint so it'll be properly spaced.

Apart from that, don't use the mongoose schema, check the schema as a middleware